extensions/sdk/setupTests.js

```diff
@@ -1,3 +1,4 @@
 process.on('unhandledRejection', (err) => {
     console.log(err);
 });
+chrome.runtime.id = "testid";
```

Comment:
@thomik-corp note that I removed sinon and switched to using jest, I feel like this is a better approach but lmk what you think.

Reply:
Yes thats indeed the right thing to do. Jest gives us all the mocking!